src/main/navigation/navigation_fixedwing.c

```diff
@@ -424,7 +424,7 @@ void applyFixedWingPitchRollThrottleController(navigationFSMStateFlags_t navStat
         rollCorrection += posControl.rcAdjustment[ROLL];
     }
 
-    if (isPitchAdjustmentValid && (navStateFlags & NAV_CTL_ALT)) {
+    if (isPitchAdjustmentValid && (navStateFlags & NAV_CTL_ALT) && (getMotorStatus() != MOTOR_STOPPED_USER)) {
```

Comment:
Why checking for motor-stop here? When ALTHOLD is paused NAV_CTL_ALT is not set anyway.

Reply:
True ! It is a reminiscence of old code. Fixed